src/microbe_stage/editor/tooltips/ModifierInfoLabel.cs

```diff
@@ -128,6 +129,12 @@ public void AdjustValueColor(float value, bool inverted = false)
         }
     }
 
+    private void AdjustSize(float size)
+    {
+        if (valueLabel != null)
+            valueLabel!.RectMinSize = new Vector2(size, 20.0f);
```

Comment:
Shouldn't this null override be completely unnecessary thanks to the if this is inside of?